play-json/jvm/src/main/scala/play/api/libs/json/jackson/JacksonJson.scala

```diff
@@ -125,6 +123,16 @@ private[jackson] class JsValueSerializer(jsonConfig: JsonConfig) extends JsonSer
       case JsNull => json.writeNull()
     }
   }
+
+  private def containsEOrDot(s: String): Boolean = {
```

Comment:
`s.exists(...)`

Reply:
Are you suggesting a rename for the the 'containsEOrDot' function or a change in its implementation?
I am not aware of a method on String called 'exists'. I checked the Javadocs.

Reply:
actually, I see your point now - I will make the change you requested